dev-lang/vala/vala-0.40.1.recipe

```diff
@@ -0,0 +1,120 @@
+SUMMARY="An object-oriented programming language that uses the GObject system"
+DESCRIPTION="Vala is a new programming language that aims to bring modern \
+programming language features to GNOME developers without imposing any \
+additional runtime requirements and without using a different ABI \
+compared to applications and libraries written in C."
+HOMEPAGE="https://wiki.gnome.org/Projects/Vala"
+COPYRIGHT="2006-2018 Jürg Billeter
+	2006-2018 Raffaele Sandrini"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://download.gnome.org/sources/vala/${portVersion%.*}/vala-$portVersion.tar.xz"
+CHECKSUM_SHA256="79b544305640b00010008cad0df1a1742363f636e4a7e3d410f651edc728f605"
+SOURCE_DIR="vala-$portVersion"
+PATCHES="vala-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 ?x86 ?x86_64"
+SECONDARY_ARCHITECTURES="?x86"
+
+libVersion="0.0.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+portVers="${portVersion%.*}"
+
+PROVIDES="
+	vala$secondaryArchSuffix = $portVersion
+	cmd:vala$secondaryArchSuffix
+	cmd:vala_$portVers$secondaryArchSuffix
+	cmd:vala_gen_introspect$secondaryArchSuffix
+	cmd:vala_gen_introspect_$portVers$secondaryArchSuffix
+	cmd:valac$secondaryArchSuffix
+	cmd:valac_$portVers$secondaryArchSuffix
+	cmd:valadoc$secondaryArchSuffix
+	cmd:valadoc_$portVers$secondaryArchSuffix
+	cmd:vapigen$secondaryArchSuffix
+	cmd:vapigen_$portVers$secondaryArchSuffix
+	lib:libvala_$portVers$secondaryArchSuffix = $libVersionCompat
+	lib:libvaladoc_$portVers$secondaryArchSuffix = $libVersionCompat
+	"
```

Comment:
```diff
 	lib:libvala_$portVers$secondaryArchSuffix = $libVersionCompat
 	lib:libvaladoc_$portVers$secondaryArchSuffix = $libVersionCompat
+	devel:libvala_$portVers$secondaryArchSuffix = $libVersionCompat
+	devel:libvaladoc_$portVers$secondaryArchSuffix = $libVersionCompat
 	"
```
This is what we did for vala `0.36.12` and I think we should do so here too.

Reply:
Why are we merging `_devel` to `vala`? I think I have missed the discussions about this.

Reply:
Sounds fine enough to me. Debian appears to split the package, but I believe it's only due to their policy to split things apart. If `vala` is not really usable without the `_devel` part, then I also believe we shouldn't split it apart.

Reply:
It was still something that has been there in the past (before we removed the _devel package for vala IIRC)

Reply:
Yep, it was there since the very first time the recipe was added. That was for vala `0.30.1`.

BTW, I'll try to build vala-0.40.3.
```diff
-CHECKSUM_SHA256="79b544305640b00010008cad0df1a1742363f636e4a7e3d410f651edc728f605"
+CHECKSUM_SHA256="6d4f1f0b2edaa0d4aa96f72735a9845e6b1acf80a0a2ae494b5d43e07833119e"
```

Reply:
I'll try it here also, shouldn't be a real change from the version at hand ...